### PR TITLE
v.1.0.4 change log

### PR DESCRIPTION
**Pull request by jarindr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +38,12 @@
 
 - docs: extend description of running cabbage tests [\#1658](https://github.com/omgnetwork/elixir-omg/pull/1658) ([pnowosie](https://github.com/pnowosie))
 
+### Untagged pull requests
+
+- feat: handle in-flight exits deletions [\#1701](https://github.com/omgnetwork/elixir-omg/pull/1701) ([pgebal](https://github.com/pgebal))
+- add feefeed docker to elixir-omg setup [\#1700](https://github.com/omgnetwork/elixir-omg/pull/1700) ([boolafish](https://github.com/boolafish))
```
Comment (boolafish) [inline CHANGELOG.md:44]: I just tagged it 😛

Comment (jarindr) [reply CHANGELOG.md]: regenerated :trollface:

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +39,11 @@
 
 - docs: extend description of running cabbage tests [\#1658](https://github.com/omgnetwork/elixir-omg/pull/1658) ([pnowosie](https://github.com/pnowosie))
 
+### Untagged pull requests
+
+- feat: handle in-flight exits deletions [\#1701](https://github.com/omgnetwork/elixir-omg/pull/1701) ([pgebal](https://github.com/pgebal))
```
Comment (unnawut) [inline CHANGELOG.md:44]: Can you label these two PRs and regenerate the changelog?

File: CHANGELOG.md (hunk 1 above)
Comment (unnawut) [reply CHANGELOG.md]: dammit you two

File: CHANGELOG.md (hunk 2 above)
Comment (unnawut) [reply CHANGELOG.md:44]: meh

**Review by boolafish — APPROVED** [review]
@unnawut "[meh](https://github.com/omgnetwork/elixir-omg/pull/1719#discussion_r482013286)"ed, so I approve!

**Review by unnawut — APPROVED** [review]
> @unnawut "[meh](https://github.com/omgnetwork/elixir-omg/pull/1719#discussion_r482013286)"ed, so I approve!

:joy: Lol, looks like two more untagged PRs left though, @jarindr can you help tag & regenerate them before merging? Thx!

**Comment by unnawut** [comment]
Perfect thanks!

**Pull request merged**